content/docs/hooks-custom.md

```diff
@@ -184,24 +183,24 @@ function ChatRecipientPicker() {
 }
 ```
 
-We keep the currently chosen friend ID in the `recipientID` state variable, and update it if the user chooses a different friend in the `<select>` picker.
+Mantenemos el amigo seleccionado actual en la variable de estado `recipientID`, y la actualizamos si el usuario elige a un amigo diferente en el selector `<select>`.
 
-Because the `useState` Hook call gives us the latest value of the `recipientID` state variable, we can pass it to our custom `useFriendStatus` Hook as an argument:
+Como la llamada al Hook `useState` nos da el ultimo valor de la variable de estado `recipientID`, podemos pasarla a nuestro Hook personalizado `useFriendStatus` como un argumento:
 
 ```js
   const [recipientID, setRecipientID] = useState(1);
   const isRecipientOnline = useFriendStatus(recipientID);
 ```
 
-This lets us know whether the *currently selected* friend is online. If we pick a different friend and update the `recipientID` state variable, our `useFriendStatus` Hook will unsubscribe from the previously selected friend, and subscribe to the status of the newly selected one.
+Esto nos permite saber cuando el amigo* actualmente seleccionado* esta en linea. Si elegimos un amigo diferente y actualizamos la variable de estado `recipientID`, nuestro Hook `useFriendStatus` va a desuscribirse de el amigo previamente seleccionado, y suscribirse al estado de el nuevo seleccionado.
```

Comment:
```suggestion
Esto nos permite saber cuando el amigo* actualmente seleccionado* esta en línea. Si elegimos un amigo diferente y actualizamos la variable de estado `recipientID`, nuestro Hook `useFriendStatus` eliminará su suscripción del amigo previamente seleccionado, y suscribirse al estado del nuevo seleccionado.
```

Reply:
I believe it's `está` in "Está en linea".

And for the ending of the last sentence:
> y se suscribirá al estado del nuevo seleccionado.